app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="sc-TakeoverHeader-content">
+  <figure class="sc-TakeoverHeader-logoContainer sc-p--xs">
+    <%= link_to(image_tag('ttlogo.png', size: "47x47"), admin_root_url) %>
+  </figure>
+  <% if signed_in? %>
+    <div class="sc-TakeoverHeader--multipleCta">
```

Comment:
In my comments above, I didn't realize this modifier selector wasn't on the component root element. Sticking with the component usage guidelines, we should end up with markup like this:

```erb
<div class="sc-TakeoverHeader-cta">
  <%= link_to 'Splits', admin_root_path, class: 'sc-p--s' %>
  <%= link_to 'Log out', destroy_admin_session_path, method: :delete. class: 'sc-p--s' %>
</div>
```